projects/dotcms-field-elements/src/components/dot-tags/components/dot-autocomplete/dot-autocomplete.e2e.ts

```diff
@@ -4,197 +4,230 @@ describe('dot-autocomplete', () => {
+                        </span>
+                        ult-2
+                    </li>
+                    <li class="autoComplete_result" data-result="result-3" tabindex="1">
+                        <span class="autoComplete_highlighted">
+                            res
+                        </span>
+                        ult-3
+                    </li>
+                    <li class="autoComplete_result" data-result="result-4" tabindex="1">
+                        <span class="autoComplete_highlighted">
+                            res
+                        </span>
+                        ult-4
+                    </li>
+                    <li class="autoComplete_result" data-result="result-5" tabindex="1">
+                        <span class="autoComplete_highlighted">
+                            res
+                        </span>
+                        ult-5
+                    </li>
+                `);
             });
 
-            it('should not broke when threshold is not a number', async () => {
-                element.setAttribute('threshold', {});
+            it('should not show results before when meet', async () => {
+                input.type('re');
                 await page.waitForChanges();
 
-                expect(await element.find('input')).toBeDefined();
+                const ul = await element.find('ul');
+                expect(ul.innerHTML).toBe('');
             });
+        });
 
-            it('should not broke when debounce is not a number', async () => {
-                element.setAttribute('debounce', {});
+        describe('maxResults', () => {
+            let input;
+
+            it('should show 5 (default) results', async () => {
+                input = await page.find('input');
+                input.type('res');
                 await page.waitForChanges();
 
-                expect(await element.find('input')).toBeDefined();
+                const lis = await element.findAll('ul li');
+                expect(lis.length).toBe(5);
             });
 
-            it('should not broke when maxResults is not a number', async () => {
-                element.setAttribute('maxResults', {});
+            it('should show 3 results', async () => {
+                element.setAttribute('max-results', 3);
                 await page.waitForChanges();
+                input = await page.find('input');
 
-                expect(await element.find('input')).toBeDefined();
-            });
-
-            it('should not broke when data does is not a function', async () => {
-                element.setAttribute('data', {});
+                input.type('res');
                 await page.waitForChanges();
 
-                expect(await element.find('input')).toBeDefined();
+                const lis = await element.findAll('ul li');
+                expect(lis.length).toBe(3);
             });
         });
+
+        xdescribe('debounce', () => {});
+
+        xdescribe('data', () => {});
```

Comment:
I think we already have this test here

https://github.com/dotCMS/core-web/blob/175c864cebb2bd72595067c141a165387bed6053/projects/dotcms-field-elements/src/components/dot-tags/components/dot-autocomplete/dot-autocomplete.e2e.ts#L74

Reply:
Well... I guess, but I was trying to test passing `data` but is not possible, when you pass the function it just doesn't do it, I tried as property and nothing.